brainstormforce/ultimate-addons-for-gutenberg
# fix: Remove option to deactivate Info Box Block & handled backward compatibility for the same.

### Description
 - Trello task - https://trello.com/c/npBgSX79/288-fix-blocks-throwing-an-error-when-activate-only-forms-or-multi-buttons

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
 - Deactivate Info Box in master.
 - Switch to the current branch.
 - Refresh in the Editor 2 times.
 - Info Box should work.

### Checklist:
